src/plone/restapi/tests/http-examples/transactions_revert.req

```diff
@@ -0,0 +1,9 @@
+PATCH /plone/@transactions HTTP/1.1
+Accept: application/json
+Authorization: Basic YWRtaW46c2VjcmV0
+Content-Type: application/json
+
+{
+    // Single or more Transaction_Id can be inserted in the List
+    "Transactions_ids": ["Transactions_ids","Transactions_ids"]
```

Comment:
Better to add a "real" example here so the documentation is more clear. Also should the variable name be "transaction_ids" (all lower case).